### ISSUE TITLE
Unable to locate or obtain driver for chrome

### ISSUE THREAD

**Issue by jonathanrstern** [heroku/heroku-buildpack-chromedriver#54]
Message: Unable to locate or obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location

Was working yesterday. Now failing today.

GOOGLE_CHROME_BIN and CHROMEDRIVER_PATH are both set.

```
chrome_options = webdriver.ChromeOptions()
chrome_options.add_argument("--headless")
chrome_options.add_argument("--disable-gpu")
chrome_options.add_argument("--no-sandbox")
chrome_options.add_argument("--disable-dev-shm-usage")

chrome_options.binary_location = os.environ.get("GOOGLE_CHROME_BIN")
service = Service(executable_path=os.environ.get("CHROMEDRIVER_PATH"))
browser = webdriver.Chrome(service=service, options=chrome_options)
```

**Comment by ghost** [comment]
Can you try without explicitly pass the `executable_path`
Just 
`service=Service()`

**Comment by jonathanrstern** [comment]
Does chromedriver no longer need a specified executable path on Heroku?

Do I need chromedriver_autoinstaller or ChromeDriverManager in order for this to work?

**Comment by ghost** [comment]
No the path is automatically set, no need to explicitly defined

**Comment by jonathanrstern** [comment]
Excellent, just pushed an update... will let you know if it works!

UPDATE: it worked!!!

**Comment by edmorley** [comment]
Support for newer chromedriver versions was fixed by #51 + #52.

Note: Older versions may no longer be available from the new download site. If you see errors, try unsetting `CHROMEDRIVER_VERSION` to use the default latest version.

**Comment by jonathanrstern** [comment]
Once again, out of the blue, Heroku Chrome has stopped working. Again, the culprit: Chrome auto-updates.

Is there anything I can do?

```
chrome_options = webdriver.ChromeOptions()
chrome_options.add_argument("--headless")
chrome_options.add_argument("--disable-gpu")
chrome_options.add_argument("--no-sandbox")
chrome_options.add_argument("--disable-dev-shm-usage")

browser = webdriver.Chrome(options=chrome_options)
```
